sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -274,15 +249,20 @@ def __repr__(self) -> str:
 
 
 class TablePropertiesPaged(PageIterator):
-    """An iterable of Table properties.
+    """An iterable of Table properties."""
 
-    :param callable command: Function to retrieve the next page of items.
-    :keyword int results_per_page: The maximum number of results retrieved per API call.
-    :keyword str filter: The filter to apply to results.
-    :keyword str continuation_token: An opaque continuation token.
-    """
+    results_per_page: Optional[int]
+    """The maximum number of results retrieved per API call."""
+    filter: Optional[str]
+    """The filter to apply to results."""
 
-    def __init__(self, command, **kwargs):
+    def __init__(self, command: Callable, **kwargs: Any) -> None:
+        """
+        :param callable command: Function to retrieve the next page of items.
+        :keyword int results_per_page: The maximum number of results retrieved per API call.
+        :keyword str filter: The filter to apply to results.
+        :keyword str continuation_token: An opaque continuation token.
```

Comment:
We probably don't need to publicly document the keyword-args here - as customers should never construct this.
Or at least, we can add a line that explicitly states:
"This object should not be constructed directly, instead it is returned from TableServiceClient.list_tables"

Reply:
Removed the docstring